src/unix/aix.c

```diff
@@ -1142,7 +1142,7 @@ int uv_interface_addresses(uv_interface_address_t** addresses,
     memcpy(flg.ifr_name, p->ifr_name, sizeof(flg.ifr_name));
     if (ioctl(sockfd, SIOCGIFFLAGS, &flg) == -1) {
       uv__close(sockfd);
```

Comment:
I think the calls to uv__close should be written as `SAVE_ERRNO(uv__close(sockfd))` now.